libraries/ms-common/src/stm32f0xx/i2c.c

```diff
@@ -1,5 +1,6 @@
 #include "i2c.h"
 #include <stdbool.h>
+#include "critical_section.h"
```

Comment:
I haven't had the chance to test this but assuming the cleanup works properly, this is the same change I made in #194 except neater.

Reply:
Please test when you have a chance :)